src/labone/core/value.py

```diff
@@ -322,3 +324,81 @@ def _value_from_python_types(
             msg,
         )
     return request_value
+
+
+def _value_from_python_types_dict(
```

Comment:
issue: This should not be a private function ... its used outside of this submodule ... so its not private 😇